modules/nextflow/src/main/groovy/nextflow/processor/PublishDir.groovy

```diff
@@ -509,13 +516,14 @@ class PublishDir {
             FilesEx.mklink(source, [hard:true], destination)
         }
         else if( mode == Mode.MOVE ) {
-            FileHelper.movePath(source, destination)
+            FileHelper.movePath(source, destination, options as CopyOption[])
         }
         else if( mode == Mode.COPY ) {
-            FileHelper.copyPath(source, destination)
+            FileHelper.copyPath(source, destination, options as CopyOption[])
         }
         else if( mode == Mode.COPY_NO_FOLLOW ) {
-            FileHelper.copyPath(source, destination, LinkOption.NOFOLLOW_LINKS)
+            options.add(LinkOption.NOFOLLOW_LINKS)
+            FileHelper.copyPath(source, destination, options as CopyOption[])
```

Comment:
Could make sense to refactor this as another config option called `workflow.output.followLinks` (default true) instead of a separate publish mode

Reply:
Maybe, mostly interested to close this tho